mappings/net/minecraft/client/render/entity/model/AbstractZombieModel.mapping

```diff
@@ -1,3 +1,3 @@
 CLASS net/minecraft/unmapped/C_eugfztfg net/minecraft/client/render/entity/model/AbstractZombieModel
-	METHOD m_fidwubpx isAttacking (Lnet/minecraft/unmapped/C_hlqarogo;)Z
+	METHOD m_fidwubpx isAggressive (Lnet/minecraft/unmapped/C_hlqarogo;)Z
```

Comment:
Reason for this change? I mean when a zombie is aggressive its attacking so I guess this is generally fine, just found it a bit odd.

Reply:
Attacking implies the act of an attack, where aggressive is part of the state of the zombie ai, enough of a different to be worth while IMO

Reply:
yeah, you are correct.